src/components/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import "./LandingPage.scss";
+import LoginButton from "../Login/LoginButton";
+import Button from "../Button/Button";
+import startIcon from "../../assets/start_icon.svg";
+import { FileIconHtml, FileIconPython } from "../../Icons";
+
+const LandingPage = () => {
+  const { t, i18n } = useTranslation();
+  const locale = i18n.language;
```

Comment:
🔧 `isDarkMode` hook is needed here to toggle the `startIcon` between dark and light versions